style.css

```diff
@@ -0,0 +1,12 @@
+
+.bord {
+    background-color: rgba(197, 194, 194, 0.4);
```

Comment:
I think it is best to use 2 white spaces as indentation. 
Otherwise, Linters will detect it as an error.

Reply:
- Good work here